src/docs/Components/DataTable/index.mdx

```diff
@@ -121,7 +121,7 @@ Depending on the context, the column header can be fixed as well.
 
 #### Mass selection
 
-Once users select a row, a dedicated banner for mass selection will appear, allowing them to select all rows in the table, including all pages.
+Once users select all rows of a page, a dedicated banner for mass selection will appear, allowing them to select all rows in the table, including all pages.
```

Comment:
~~Once users select all rows of a page~~
Once users have selected all the rows on a page,